DemoApp/BannerShowingConnectionDelegate.swift

```diff
@@ -34,6 +34,7 @@ extension BannerShowingConnectionDelegate: ChatConnectionControllerDelegate {
             bannerView.update(text: "Disconnecting...")
         case .connecting:
             bannerView.update(text: "Connecting...")
+            showBanner()
```

Comment:
This always displays "Connecting..." when you login :/ Should we for now just discard this change and wait for channel list states implementation?

Reply:
> This always displays "Connecting..." when you login

Showing `Connecting...` when the user logs in online is expected. However, when launching the app offline, showing `connecting` when nothing really happens in LLC is confusing. That's probably smth we might wanna invest some time into (cc @polqf)

> Should we for now just discard this change and wait for channel list states implementation?
The user should at least be aware, that the chat is not connected. Showing `connecting` is better than nothing, IMO.  If we want better UX we could show the connection state in channel list navigation subtitle like Telegram to make the connection state look like some small bit of info but not a warning (that annoys by overlapping the channel list)

Reply:
Like we agreed on Slack, we would not do anything, not show any alert until we tackle the channel states. IMO I rather not have any feedback for now when your connection is Offline than have a "Conecting..." appearing every time I log in with a user.

Reply:
The goal of this PR is to "When you open the channel list while offline, the data should be there, and should not present any alert". This is fine. It is just the demo app, I would not overthink it and make it perfect in this PR.

Reply:
We agreed to have a banner, didn't we? 🤔